onchain/rollups/.changeset/blue-schools-think.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
this changeset file has nothing in it?

Reply:
Yes, this is necessary because otherwise, when I run...

```
$ yarn changeset status
🦋  error Some packages have been changed but no changesets were found. Run `changeset add` to resolve this error.
🦋  error If this change doesn't need a release, run `changeset add --empty`.
```